src/main/java/core/basesyntax/FileEditor.java

```diff
@@ -0,0 +1,121 @@
+package core.basesyntax;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.FileAlreadyExistsException;
+import java.nio.file.Files;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.attribute.FileTime;
+import java.util.List;
+import java.util.Scanner;
+
+public class FileEditor {
+    public void createFile(String fileName, String inputPath) {
+        Path path = Paths.get(inputPath + fileName);
+        try {
+            Files.createFile(path);
+            System.out.println("Successfully creation!");
+        } catch (FileAlreadyExistsException e) {
```

Comment:
don't use try catch to build logic. You can use Files.exists() and if construction instead of try catch here  for check